src/rhub/api/bare_metal/host.py

```diff
@@ -38,6 +28,18 @@ def host_create(body):
     return host.to_dict_with_super()
 
 
+def host_create_ipmi(body):
```

Comment:
There are other hardware drivers that I listed in the HW wish list.  Of course we need equipment to add that code.